webpack.config.js

```diff
@@ -0,0 +1,11 @@
+module.exports = {
+	mode: 'production',
+	entry: './index.js',
+	output: {
+		filename: 'index.js',
+		library: {
+			type: 'commonjs2',
+		},
+	},
+	devtool: 'source-map',
+};
```

Comment:
Maybe you want me to use babel to get a browser bundle supported by more browsers?